### PR TITLE
feat: improve PHP config

### PR DESCRIPTION
**Pull request by dunglas**
Tweak PHP config according to Blackfire's recommendation.

### CODE REVIEW

File: docker/php/conf.d/symfony.dev.ini [hunk 1]
```diff
@@ -1,11 +1,14 @@
 apc.enable_cli = 1
 date.timezone = UTC
-session.auto_start = Off
-short_open_tag = Off
```
Comment (dunglas) [inline docker/php/conf.d/symfony.dev.ini:4]: They already are the defaults: https://github.com/php/php-src/blob/master/php.ini-development

File: docker/php/conf.d/symfony.dev.ini [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
 # http://symfony.com/doc/current/performance.html
 opcache.interned_strings_buffer = 16
 opcache.max_accelerated_files = 20000
 opcache.memory_consumption = 256
+enable_file_override = 1
```
Comment (dunglas) [inline docker/php/conf.d/symfony.dev.ini:8]: According to Blackfire, it's better to enable this and to disable timestamp validation.

File: docker/php/conf.d/symfony.prod.ini [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 opcache.max_accelerated_files = 20000
 opcache.memory_consumption = 256
-opcache.validate_timestamps = 0
-realpath_cache_size = 4096K
-realpath_cache_ttl = 600
-opcache.preload_user = www-data
-opcache.preload = /srv/app/config/preload.php
+opcache.enable_file_override = 1
+opcache.preload_user=www-data
+opcache.preload=/srv/api/Oconfig/preload.php
```
Comment (jdreesen) [inline docker/php/conf.d/symfony.prod.ini:15]: Is this a typo?

```suggestion
opcache.preload=/srv/api/config/preload.php
```

File: docker/php/docker-entrypoint.sh [hunk 4]
```diff
@@ -7,11 +7,9 @@ if [ "${1#-}" != "$1" ]; then
 fi
 
 if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
-	PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-production"
 	if [ "$APP_ENV" != 'prod' ]; then
-		PHP_INI_RECOMMENDED="$PHP_INI_DIR/php.ini-development"
+		ln -sf "$PHP_INI_RECOMMENDED" "$PHP_INI_DIR/php.ini-development"
```
Comment (maxhelias) [inline docker/php/docker-entrypoint.sh:11]: Shouldn't it be more like this?
```suggestion
		ln -sf "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini"
```

Comment (dunglas) [reply docker/php/docker-entrypoint.sh:11]: Ooops ^^

**Review by maxhelias — APPROVED** [review]

**Pull request merged**